### PR TITLE
LA- 286 Duplicate document

### PR DESCRIPTION
**Pull request by imGok**
duplicatedocuemnts

### CODE REVIEW

**Comment by imGok** [comment]
Review, don't merge please

**Review by hoangdat — APPROVED** [review]

File: lib/presentation/widget/myspace/my_space_viewmodel.dart [hunk 1]
```diff
@@ -328,6 +330,27 @@ class MySpaceViewModel extends BaseViewModel {
...
+      cancelSelection();
+    }
+
+    store.dispatch(_duplicateAction(documents));
+  }
+
+  OnlineThunkAction _duplicateAction(List<Document> documents) {
+    return OnlineThunkAction((Store<AppState> store) async {
+      await _duplicateMultipleFilesInteractor
+          .execute(documents: documents)
```
Comment (dab246) [inline lib/presentation/widget/myspace/my_space_viewmodel.dart:345]: Indent should use 2 spaces

File: lib/presentation/widget/myspace/my_space_viewmodel.dart [hunk 2]
```diff
@@ -328,6 +330,27 @@ class MySpaceViewModel extends BaseViewModel {
...
+          .then((result) => result.fold(
+              (failure) => store.dispatch(MySpaceAction(Left(failure))),
```
Comment (dab246) [inline lib/presentation/widget/myspace/my_space_viewmodel.dart:347]: same

**Review by dab246 — APPROVED** [review]
LGTM

**Review by hoangdat — APPROVED** [review]

**Pull request merged**